### DISCUSSION TITLE
Validação de formulário (Fundamentos do Blazor com .net 8)

### DISCUSSION THREAD

**Discussion by Beto-D3v** [balta-io/forum#440] [Tira Dúvidas] [answered]
Opa pessoal, bom dia.

Estou seguindo o curso de fundamentos do Blazor com o .Net 8 e criei o projeto como Blazor Server com sugerido nas aulas iniciais, porém, ao colocar a diretiva @attribute [RenderModeInteractiveServer] , dentro da página create.razor, esta diretiva não está sendo localizada. Será que preciso adicionar algo no program ou no imports razor?

Sem essa diretiva, quando clico no botão submit, ele acaba fazendo o post para o servidor, ao invés de ter o comportamento da aula de testar as validações. Já testei inclusive a diretiva @rendermode InteractiveServer , mas não funcionou.

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @Beto-D3v, como vai?

Conforme mostrado na primeira aula do curso, houve uma mudança na versão final do Blazor onde essa diretiva foi trocada por `@rendermode InteractiveServer`.

Segue o link da aula:
https://balta.io/cursos/fundamentos-blazor-dotnet-8/3002M00A01

**Reply by Beto-D3v** [reply]
Opa, muito obrigado, Balta!

Vou mudar a diretiva aqui.
